recipes/ipfshttpclient/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ipfshttpclient" %}
+{% set version = "0.6.0.post1" %}
```

Comment:
Is there is a fully released version that you can use here?

Reply:
It was released normally... not sure what post means. Shortly after this recipe was made, they put out 0.6.1 so I updated it to that.

Reply:
Ahh yeah, I mean a release without the `post1`